src/main.js

```diff
@@ -4,3 +4,6 @@ export default placeholder;
 
 // Dialogs
 export { default as ModalDialog } from './components/ModalDialog';
+
+// Plugins
+export { default as Nvw } from './plugins';
```

Comment:
can you remove this export statement from main.js and import this plugin where needed?

Reply:
> can you remove this export statement from main.js and import this plugin where needed?

Fixed.